modular_zubbers/code/modules/research/designs/misc_designs.dm

```diff
@@ -7,3 +7,27 @@
 	build_path = /obj/item/disk/plantgene
 	category = list("Electronics")
 	departmental_flags = DEPARTMENT_BITFLAG_SERVICE
+
+/datum/design/nitrogen_tank
+	name = "Nitrogen Tank"
+	desc = "An empty nitrogen tank."
+	id = "nitrogen_tank"
+	build_type = PROTOLATHE | AWAY_LATHE
+	materials = list(/datum/material/iron =SHEET_MATERIAL_AMOUNT)
+	build_path = /obj/item/tank/internals/nitrogen
```

Comment:
This is not the empty version. If it doesn't exist, create it.

Reply:
it is. the empty version is internals/nitrogen and the full one is internals/nitrogen/full because there's no consistency

Reply:
![image](https://github.com/user-attachments/assets/16f76d1d-8896-4054-9f62-8cf2c1bdb088)

Reply:
this is in fact, a tank with gas

Reply:
you're telling me the gas tank I spawn in local doesn't have gas in it, but the one in the code does. this is insanity of the highest order